packages/@aws-cdk/service-spec-types/src/types/database.ts

```diff
@@ -21,13 +21,15 @@ import {
   RegionHasResource,
   RegionHasService,
   UsesType,
+  ResourceScrutinyType,
 } from './resource';
 
 export function emptyDatabase() {
   return new Database(
     {
       resource: entityCollection<Resource>().index({
         cloudFormationType: fieldIndex('cloudFormationType', stringCmp),
+        scrutinizable: fieldIndexWithDefault('scrutinizable', stringCmp, ResourceScrutinyType.None),
```

Comment:
Ended up doing this instead of using an index that can understand `undefined` because querying `db.lookup('resource', 'scrutinizable', 'equals', ResourceScrutinyType.None)` is nicer.